### PR TITLE
 refactor bullet, change velocity of bullet

### PR DESCRIPTION
**Pull request by alvarogerardo007**

### CODE REVIEW

**Comment by Dru27** [comment]
Good job!

**Review by Dru27 — APPROVED** [review]

**Review by adrianovi — COMMENTED** [review]
The speed of the bullet is increasing, right?

File: src/main/java/org/fundacionjala/at15/spaceinvaders/Bullet.java [hunk 1]
```diff
@@ -16,7 +15,7 @@ public Bullet(int posx, int posy, boolean fire) {
     }
     public int getPosx() {
 
-        return posStaticx;
+        return posx;
```
Comment (adhes14) [inline src/main/java/org/fundacionjala/at15/spaceinvaders/Bullet.java:18]: Good choice

**Review by adhes14 — APPROVED** [review]
Nice work

**Pull request merged**